src/utils/aiService.js

```
@@ -0,0 +1,479 @@
+                // Use the endpoint directly from centralized config (already includes /messages)
+                url = providerConfig.endpoint || 'https://api.anthropic.com/v1/messages';
+                headers = {
+                    'x-api-key': apiKey,
+                    'Content-Type': 'application/json',
+                    'anthropic-version': '2023-06-01'
+                };
+                break;
+
+            case 'google':
+                // Use endpoint from centralized config and replace {model} placeholder
+                const endpoint = providerConfig.endpoint || 'https://generativelanguage.googleapis.com/v1beta/models/{model}:generateContent';
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)